packages/theme/src/Components/TextField/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ForwardedRef, forwardRef } from 'react'
+import {
+    Box,
+    formHelperTextClasses,
+    makeStyles,
+    TextField,
+    StandardTextFieldProps,
+    InputProps,
+    Typography,
+} from '@material-ui/core'
+import { getMaskColor } from '../../constants'
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        width: '100%',
+        // marginBottom: theme.spacing(4),
+    },
+    label: {
+        fontSize: 12,
+        lineHeight: '16px',
+        fontWeight: 'bolder',
+    },
+    field: {
+        width: '100%',
+        [`& > .${formHelperTextClasses.root}`]: {
+            paddingLeft: theme.spacing(0.5),
+            borderLeft: 'solid 2px',
+            borderRadius: '2px',
+            fontSize: 12,
+            lineHeight: '16px',
+        },
+    },
+    input: {
+        padding: theme.spacing(1),
+        background:
+            theme.palette.mode === 'dark' ? getMaskColor(theme).lightBackground : getMaskColor(theme).normalBackground,
+        fontSize: 12,
+        lineHeight: '16px',
+    },
+}))
+
+type MaskTextFieldProps = Exclude<StandardTextFieldProps, 'variant'>
+
+export const MaskTextField = forwardRef((props: MaskTextFieldProps, ref: ForwardedRef<any>) => {
```

Comment:
I suggest to tweak it via the theme system. By adding a new variant.

Reply:
I think it's fine as a component, by adding a new variant it leads to complications and may not achieve part of the UI design.